ktor-client/ktor-client-curl/build.gradle

```diff
@@ -23,7 +23,8 @@ kotlin {
                                 '/usr/include/curl',
                                 '/usr/include/x86_64-linux-gnu/curl',
                                 '/usr/local/Cellar/curl/7.62.0/include/curl',
-                                '/usr/local/Cellar/curl/7.63.0/include/curl'
+                                '/usr/local/Cellar/curl/7.63.0/include/curl',
+                                '/usr/local/Cellar/curl/7.65.3/include/curl'
```

Comment:
Can't we collect a list of include directories programmatically? Something like `new File("/usr/local/Cellar/curl/").listFiles()?.collect { new File(it, "include/curl") }.findAll { it.exists() }`

Reply:
We should check how this affects publication before